expression/integration_test.go

```diff
@@ -2172,10 +2172,14 @@ func (s *testIntegrationSuite) TestBuiltin(c *C) {
 	result.Check(testkit.Rows("<nil>"))
 	result = tk.MustQuery(`select cast(cast('2017-01-01 01:01:11.12' as date) as datetime(2));`)
 	result.Check(testkit.Rows("2017-01-01 00:00:00.00"))
-
 	result = tk.MustQuery(`select cast(20170118.999 as datetime);`)
 	result.Check(testkit.Rows("2017-01-18 00:00:00"))
 
+	tk.MustExec(`create table tb5(a bigint(64) unsigned, b double);`)
```

Comment:
why not directly create a json column?

Reply:
If directly creat json, then I can not get a uint field(https://github.com/pingcap/tidb/issues/11386#issuecomment-514594521).

Reply:
how about casting '9223372036854775808' to uint then cast to json?